app/views/container_service/_main.html.haml

```diff
@@ -1,8 +1,15 @@
 = render :partial => "layouts/flash_msg"
 .row
   .col-md-12.col-lg-6
-    = render :partial => "shared/summary/textual", :locals => { :title => _("Properties"), :items => textual_group_properties }
-    = render :partial => "shared/summary/textual_multilabel", :locals => { :title => _("Port Configurations"), :items => textual_group_port_configs }
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Properties"),
```

Comment:
@abonas as I explained to Oleg, I broke the line because of Rubocop's "LineLength: Line is too long" alert. 
I can rollback to the previous way but the warnings will keep occurring when running rubocop. Whatever you decide is fine by me.